src/utils/util.jl

```diff
@@ -350,4 +350,42 @@ function split_interval(dom::Hyperrectangle, i::Int64)
     input_upper[i] = dom.center[i] + dom.radius[i]
     input_split_right = Hyperrectangle(low = input_lower, high = input_upper)
     return (input_split_left, input_split_right)
+end
+
+
+"""
+    isfeasible(model::Model)
+Test if the termination status of `model` can be considered feasible.
+"""
+function isfeasible(model::Model)
```

Comment:
This is the crux of the change. `isfeasible`replaces checks like `termination_status(model) == OPTIMAL` to capture the cases we expect to encounter, which may be more diverse.